src/ui/public/documentation_links/documentation_links.js

```diff
@@ -1,30 +1,32 @@
 import { metadata } from '../metadata';
 
-const urlVersion = metadata.branch;
-const baseUrl = 'https://www.elastic.co/';
+export const DOC_LINK_VERSION = metadata.branch;
+export const ELASTIC_WEBSITE_HOST = 'https://www.elastic.co/';
```

Comment:
I'm not sure that calling this `ELASTIC_WEBSITE_HOST` provides that much additional clarity than the original name of `BASE_URL`. Additionally, I'm afraid that using the term "host" in this context could become confused with "host" portion of the URI: `scheme:[//[user[:password]@]host[:port]][/path][?query][#fragment]`, which it is not in this scenario.